EpikCord/ext/tasks.py

```diff
@@ -51,8 +47,8 @@ def add_task(
         self,
         task: Callable[..., None],
         interval: Optional[int] = None,
-        start: Optional[float] = None,
-        until: Optional[float] = None,
+        start: Optional[datetime.timedelta] = None,
+        until: Optional[datetime.timedelta] = None,
```

Comment:
Passing timedeltas type....? Why

Reply:
if i misunderstood your former change request?

Reply:
yeah...

Reply:
i did, what was it meant to be